### PR TITLE
tests : remove lib_wallet_api tests;

### PR DESCRIPTION
**Pull request by guzzijones**

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -109,14 +109,3 @@ add_test(
...
-    coretests
-    difficulty
-    hash
-    performance_tests
-    core_proxy
-    fuzz
-    unit_tests)
-
-add_custom_target(tests DEPENDS enabled_tests)
-set_property(TARGET tests PROPERTY FOLDER "${folder}")
```
Comment (radfish) [inline tests/CMakeLists.txt]: This is still needed. The only change would be to replace `...DEPENDS enabled_tests...` with `DEPENDS coretests difficulty .... unit_tests)`. But, even this needs testing, I'm not sure if `enabled_tests` has some special meaning (not likely).

File: Makefile [hunk 2]
```diff
@@ -35,9 +35,13 @@ cmake-debug:
 debug: cmake-debug
 	cd build/debug && $(MAKE)
 
+# Temporarily disable some a tests:
+#  * coretests takes too long (~25000s)
+#  * libwallet_api_tests fail (Issue #895)
+#  * unit_tests were run separately above
```
Comment (radfish) [inline Makefile]: remove this third bullet point

File: Makefile [hunk 3]
```diff
@@ -35,9 +35,13 @@ cmake-debug:
...
 debug-test:
 	mkdir -p build/debug
-	cd build/debug && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Debug ../.. && $(MAKE) && $(MAKE) test
+	cd build/debug && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Debug ../.. &&  $(MAKE) && $(MAKE) ARGS="-E coretests\|libwallet_api_tests" test
```
Comment (radfish) [inline Makefile]: remove the backslash (no linee continuation here) and add single quotes (to make sure absolutely shell doesn't interpret the pipe) like so: `... ARGS="-E 'coretests|lib_wallet_api....'"  ...`

**Comment by fluffypony** [comment]
@guzzijones plz see feedback from moneromooo

**Comment by fluffypony** [comment]
@guzzijones still waiting on you to check feedback from moneromooo

File: tests/CMakeLists.txt (hunk 1 above)
Comment (guzzijones) [reply tests/CMakeLists.txt]: I just undid this change. I am not 100 % sure what tests it is dependent on at this point.  It was an unrelated change anyhow and should be in a separate commit.

File: Makefile (hunk 2 above)
Comment (guzzijones) [reply Makefile]: removed 1 and 3

File: Makefile (hunk 3 above)
Comment (guzzijones) [reply Makefile]: coretests removed. no pipe needed.  backslash removed.

**Comment by guzzijones** [comment]
coretests added back in.

File: tests/CMakeLists.txt [hunk 4]
```diff
@@ -121,5 +123,8 @@ set(enabled_tests
     fuzz
     unit_tests)
 
+=======
+>>>>>>> 060c3f7... squash
```
Comment (moneromooo-monero) [inline tests/CMakeLists.txt]: That's a merge conflict. This whole file can be removed from the patch.

Comment (guzzijones) [reply tests/CMakeLists.txt]: ugh lazy. my bad.

File: Makefile [hunk 5]
```diff
@@ -35,9 +35,11 @@ cmake-debug:
 debug: cmake-debug
 	cd build/debug && $(MAKE)
 
+# Temporarily disable some a tests:
```
Comment (moneromooo-monero) [inline Makefile]: "some tests"

**Pull request merged**